examples/example-ios/iosBleSdkTestApp/iosBleSdkTestApp/Views/UiHelper.swift

```diff
@@ -19,6 +19,10 @@ func getShortNameForDataType(_ feature: PolarDeviceDataType) -> String {
         return "MAG"
     case .hr:
         return "HR"
+    case .temperature:
+        return "TEMP"
+    case .pressure:
+        return "PRESS"
```

Comment:
PolarBleSdkManager is has similar cases returning strings for data types. When you add them, please keep them consistent to strings used here.